backend/inspirehep/records/marshmallow/literature/ui.py

```diff
@@ -210,6 +211,25 @@ def get_citation_pdf_urls(self, data):
                 urls.append(url)
         return urls or missing
 
+    def get_datasets(self, data):
+        dataset_links = []
+        dataset_schema_to_url_link_prefix_map = {
```

Comment:
`dataset_schema_to_url_link_prefix_map` and `dataset_schema_to_description_map` should be moved on the top of the file (uppercase).